src/coord/Axis.ts

```diff
@@ -193,6 +193,13 @@ class Axis {
             this, ticksCoords, alignWithLabel, opt.clamp
         );
 
+        if (tickModel.get('showMinLine') === false) {
```

Comment:
Modifying the result here will affect the display of labels and ticks lines. If only splitLine will be affected, it's better to do it in the render function https://github.com/apache/echarts/blob/cc0ebd2845781021cb05287fded7c94b26174619/src/component/axis/CartesianAxisView.ts#L141

Reply:
done.